go/client/kvstore_api_doc.go

```diff
@@ -2,13 +2,18 @@ package client
 
 const kvStoreAPIDoc = `"keybase kvstore api" provides a JSON API to fast, encrypted key-value storage. The "entryKey" and "namespace" fields are visible to the Keybase servers, "entryValue" is encrypted with the per-team-key. This is still a bit of a work in progress.
 
+If team is not included, or is "", or is equal to the logged-in user, then the command defaults to using the logged-in user's implicit self-team (only your user can see and decrypt this).
```

Comment:
i'm not sure it's worth having all of these options. in general, we don't want to be too flexible to accept anything at all, or people will write sloppy integrations and it will be harder for us to change things in the future. i think i'd argue the only things that should be coming into the `team` field are legit actual teams (implicit or normal). and if there isn't one at all (i.e. the field is not included) then we default to the current user's implicit self-team. an empty string should throw an error that it's not a legit team. same for writing a single username in the team field. 

and in general, if possible, we should be nudging people towards not including the team at all. like here's this cool storage you can use with namespaces and entryKeys. and it even works with teams. flipped around.

Reply:
now not accepting team names that are "" or "<yourusername>"